market_env/environment.py

```diff
@@ -1,16 +1,20 @@
 # pylint: disable=missing-module-docstring
 # pylint: disable=missing-class-docstring
 # pylint: disable=missing-function-docstring
+# pylint: disable=R0913
+# pylint: disable=too-few-public-methods
 
-from __future__ import annotations
 
+from __future__ import annotations
+from typing import List
```

Comment:
i commented on this before -- please see 
https://github.com/xujiahuayz/pbs/pull/5#discussion_r1257807995